src/com/organizationmanager/OrganizationManager.java

```diff
@@ -0,0 +1,344 @@
+                    break;
+                case "remove_greater":
+                    removeGreater(scanner);
+                    break;
+                case "history":
+                    displayHistory();
+                    break;
+                case "min_by_annual_turnover":
+                    minByAnnualTurnover();
+                    break;
+                case "count_by_type":
+                    if (argument != null) {
+                        countByType(argument);
+                    } else {
+                        System.out.println("Error: Missing type for count_by_type command.");
+                    }
+                    break;
+                case "filter_starts_with_name":
+                    if (argument != null) {
+                        filterStartsWithName(argument);
+                    } else {
+                        System.out.println("Error: Missing name for filter_starts_with_name command.");
+                    }
+                    break;
+                default:
+                    System.out.println("Unknown command. Type 'help' for a list of commands.");
+            }
+        }
+    }
+
+    private void addToHistory(String command) {
+        if (commandHistory.size() == 8) {
+            commandHistory.removeFirst();
+        }
+        commandHistory.addLast(command.split(" ", 2)[0]); // Store only the command name
+    }
+
+    private void displayHelp() {
+        System.out.println("Available commands:");
+        System.out.println("  help - Display help on available commands");
+        System.out.println("  info - Print collection information");
+        System.out.println("  show - Print all elements of the collection");
+        System.out.println("  add {element} - Add a new item to the collection");
+        System.out.println("  update id {element} - Update the value of the collection element by id");
+        System.out.println("  remove_by_id id - Remove an element from the collection by id");
+        System.out.println("  clear - Clear the collection");
+        System.out.println("  save - Save the collection to a file");
+        System.out.println("  execute_script file_name - Read and execute a script from the specified file");
+        System.out.println("  exit - Exit the program (without saving)");
+        System.out.println("  add_if_min {element} - Add a new element if its value is less than the smallest element");
+        System.out.println("  remove_greater {element} - Remove all elements greater than the specified one");
+        System.out.println("  history - Print the last 8 commands");
+        System.out.println("  min_by_annual_turnover - Output any object with the minimal annualTurnover");
+        System.out.println("  count_by_type type - Output the number of elements with the specified type");
+        System.out.println("  filter_starts_with_name name - Output elements whose name starts with the given substring");
+    }

```

Comment:
This shouldn't be hardcoded, it should depend on the available commands in the program